### ISSUE TITLE
Credentials file does not pass credentials to steamcmd login.

### ISSUE THREAD

**Issue by Aterfax** [airtonix/docker-arma3-exile#2]
Made a credentials file in .server and in server 

The login fails logging in trying to use the username: "username" despite a proper username being set.

**Comment by Aterfax** [comment]
My bad, was expecting it to pass by automatic.

Might want to add the following section to the github readme?

# How to use this image?

docker run -it \
    --env-file credentials \
    -v $PWD/arma3:/arma3 \
    -v $PWD/profiles:/profiles \
    -v $PWD/my_mission/main.cfg:/server/main.cfg \
    -v $PWD/my_mission/basic.cfg:/server/basic.cfg \
    -p 2302:2302 \
    -p 2303:2303 \
    -p 2304:2304 \
    -p 2305:2305 \
    houki/arma3

**Comment by timelf123** [comment]
@Aterfax the instructions you posted are for houki's arma image and won't link the data volume here.

Try `docker-compose up`. I'm in the process of reworking these images for 1.0.2 and part of that process is getting rid of the data container and using volumes instead, per (now) docker best practices. 

I assume @airtonix is done with Exile as he has not logged into the forum since publishing this

**Comment by Aterfax** [comment]
Yup. I noticed it not linking in the the addon or the startup lines to load it. Just meant it needs the proper startup how to command.

Was going to look into it later today / look into docker properly.
